packages/top-app-bar/index.tsx

```diff
@@ -219,6 +219,33 @@ class TopAppBar<T extends HTMLElement = HTMLHeadingElement> extends React.Compon
         }
         return 0;
       },
+      registerResizeHandler: (handler: SpecificEventListener<'resize'>) => {
```

Comment:
Can you add 2 tests for the register/deregisterResizeHandler methods please :)

Reply:
will write tomorrow